nemo/collections/asr/parts/utils/transcribe_utils.py

```diff
@@ -421,6 +420,8 @@ def transcribe_partial_audio(
                 input_signal=test_batch[0].to(device), input_signal_length=test_batch[1].to(device)
             )
             logits, logits_len = outputs[0], outputs[1]
+            if isinstance(asr_model, EncDecHybridRNNTCTCModel) and decoder_type == "ctc":
+                logits = asr_model.ctc_decoder(encoder_output=logits)
```

Comment:
Am I right in thinking that we will do a forward pass twice if it's a Hybrid model and decoder_type == "ctc"? That will essentially double the time to get a transcription, right? Could we avoid this?

Reply:
Not really. Hybrid-ctc separates encoder and decoder (the forward() function only runs encoder), so we need to run the decoder separately, while in the original ctc model the forward() has decoder included as well

Reply:
I see, thank you for explaining, LGTM